housing_django/templates/base.html

```diff
@@ -0,0 +1,66 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+      <meta name="viewport" content="initial-scale=1.0">
+        <title>Auth Demo</title>
+        <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+        <link href="{% static 'css/style.css' %}" rel="stylesheet">
+  </head>
+  <body>
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+              <span class='sr-only'>Toggle navigation</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            
+            {% if user.is_authenticated %}
+               <a class="navbar-brand" href="#">Hello, you are on your Dashboard</a>
+            {% else %}
+              <a class="navbar-brand" href="#">Hi, please login</a>
+            {% endif %}
+          </div>
+          <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav navbar-right">
+              <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >Account
+                  <span class="caret"></span>
+                </a>
+                <ul class="dropdown-menu" role="menu">
+                  <li>
+                    <a href="/home">home</a>
+                  </li>
+                  <li>
+                    <a href="#">Another action</a>
+                  </li>
+                  <li class="divider"></li>
+                  <li>
+                      <a href="#">Separated link</a>
+                  </li>
+                  <li class="divider"></li>
+                  <li>
+                      <a href="/logout">Logout</a>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+      </div>
+    </nav>
+    {% block content %}
+    {% endblock %}
+
+
+      <!-- Core Scripts - Include with every page -->
```

Comment:
For comments in the HTML, please use django commenting so that it does not show up in the final output. I.e. 
{# Core scripts - Include with every page #}